docker/docker_DB/Dockerfile

```diff
@@ -0,0 +1,16 @@
+#---------------------------------------------------------------------------
+# Dockefile to build Docker Image of application container for spring_DB running on mysql image
+#---------------------------------------------------------------------------
+
+FROM mysql:latest
+# Setting vars 
+ENV MYSQL_ALLOW_EMPTY_PASSWORD true
+ENV MYSQL_USER petclinic 
```

Comment:
It's better to reuse variable from host environment if one exists. In that case if you'll need to update DB username you will only do it once on your host system then changes would be automatically reflected on all builds and you don't need to change hardcoded values manually in every dockerfile.